dashboard/db/migrate/20160315215912_create_paired_user_levels.rb

```diff
@@ -0,0 +1,10 @@
+class CreatePairedUserLevels < ActiveRecord::Migration
+  def change
+    create_table :paired_user_levels do |t|
```

Comment:
added the indexes. `create_table` creates a primary key column `id` by default so that's the primary key here.